catalogs/HAND/build_catalog.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python3
```

Comment:
A couple flake8 linter errors in this file around spacing and unused imports. Consider running it through a flake8 + black formatter pass before merging.